pyproject.toml

```diff
@@ -7,8 +7,8 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">= 3.7, < 3.9"
-pythonnet = "2.5.2"
+python = ">= 3.7, < 3.13"
```

Comment:
Should we mention a higher version? why not py version >= 3.7?

Reply:
The pythonnet package has its supported python version as ">=3.7, < 3.13". if pythonnet 3.0.3 doesn't support the future version of Python then this will result in an error during installation.